### DISCUSSION TITLE
优先事件截图帖

### DISCUSSION THREAD

**Discussion by CHNZYX** [CHNZYX/Auto_Simulated_Universe#68] [General]
如果你有希望增加的优先事件，请在事件界面截图，保存为jpg并发送到本帖
目前存在的优先事件：阮梅，三只扑满，黑塔商店

**Comment by Eric-Joker** [comment]
emmm，新版本优先事件改成文字识别了吧。。这是我修改后的
`event_prior = ["翻开牌","等待祂","深入虫巢","选择里奥","让里奥出来吧","购买金属许愿瓶","购买超钛许愿匣","购买一个原矿盒","换取1个2星祝福","购买1个星祝福","跳上右边的砖块","丢下雕像","和序列扑满玩","信仰星神","克里珀的恩赐","哈克的藏品","动作片","感恩克里珀星神","的概率而战"]`
这里“萨里奥”连续剧是选择了里奥，也就是拿奇物换祝福那个。emm个人感觉较大多数情况下自动战斗祝福比奇物好点。
